lib/install/validate/config.go

```diff
@@ -557,3 +560,52 @@ func (v *Validator) CheckDrs(ctx context.Context) {
 	log.Info("DRS check OK on:")
 	log.Infof("  %q", v.Session.Cluster.InventoryPath)
 }
+
+// check that PersistNetworkBacking is set
+func (v *Validator) CheckPersistNetworkBacking(ctx context.Context) {
+	errMsg := "vCenter settings check SKIPPED"
+	if !v.sessionValid(errMsg) {
+		return
+	}
+	if !v.IsVC() {
+		log.Info(errMsg)
+		return
+	}
+
+	val, err := optmanager.QueryOptionValue(ctx, v.Session, persistNetworkBackingKey)
+	if err != nil {
+		// the key is not set, set it now
+		val = "false"
+	}
+	var ok bool
+	if val != "true" {
+		ok = v.SetPersistNetworkBacking(ctx)
+		if !ok {
+			log.Errorf("vCenter settings check FAILED")
+			return
+		}
+	}
+	log.Infof("vCenter settings check OK")
+}
+
+// set PersistNetworkBacking key to "true"
+func (v *Validator) SetPersistNetworkBacking(ctx context.Context) bool {
```

Comment:
This doesn't belong in the validator.

If we're setting this automatically when deploying a VCH then we should have a Dispatcher call for ConfigureVCenter or similar (my preference - automatic where viable).
If we're setting this via the `update` command then it's an explicit operation and lives with that code.

Reply:
I will move this. I haven't included work to set this through `vic-machine update`. Is it ok to open an issue to add that feature? I documented the manual steps to set this through the mob